config/locales/forms/en.yml

```diff
@@ -20,7 +20,12 @@ en:
         same level of attention as you would your password.
       subtitle: Your backup codes
       title: Backup Codes
-      warn: Download, print, or write down your personal security codes.
+      warn: Download, print, or write down your personal security codes. Your original
```

Comment:
Is the `Your original backup` text intended to be combined with another message?

Reply:
My b, cleaned up the message.